client/src/Components/Common/Navbar/Menu.js

```diff
@@ -0,0 +1,53 @@
+import React, { Component } from "react";
+import SVG from "react-inlinesvg";
+
+import { Wrapper, MenuItem } from "./Menu.style.js";
+
+// import ProfileIcon from "./../../../assets/profile-icon.svg";
+
+export default class Menu extends Component {
+  render() {
+    const { width, isLoggedIn } = this.props;
+    return (
+      <Wrapper width={width}>
+        {width < 770 && (
+          <MenuItem to="/search">
```

Comment:
lets have a constants folder with `navigationUrls.js` being a file which exports:

```
export const FAQ_URL = '/faq'
and all other react urls that are used throughout the app
etc...
```

single source of truth 🎉

Reply:
this is a great idea!